lib/cli/ui/prompt/interactive_options.rb

```diff
@@ -168,29 +243,49 @@ def process_input_until_redraw_required
         # rubocop:disable Style/WhenThen,Layout/SpaceBeforeSemicolon
         def wait_for_user_input
           char = read_char
+          @last_char = char
+
+          case char
+          when :timeout ; raise Interrupt # Timeout, use interrupt to simulate
+          when CTRL_C   ; raise Interrupt
+          end
+
           case @state
           when :root
             case char
-            when :timeout                  ; raise Interrupt # Timeout, use interrupt to simulate
             when ESC                       ; @state = :esc
             when 'k'                       ; up
             when 'j'                       ; down
-            when '0'                       ; select_n(char.to_i)
-            when ('1'..@options.size.to_s) ; select_n(char.to_i)
+            when 'e', ':', 'G'             ; start_line_select
+            when 'f', '/'                  ; start_filter
+            when ('0'..@options.size.to_s) ; select_n(char.to_i)
             when 'y', 'n'                  ; select_bool(char)
             when " ", "\r", "\n"           ; select_current  # <enter>
-            when "\u0003"                  ; raise Interrupt # Ctrl-c
```

Comment:
Ah I guess we had precedent for using UTF-8, and that makes sense anyway, since we want to deal with things in increments of 1 display with unit